src/ewatercycle/observation/grdc.py

```diff
@@ -351,12 +351,12 @@ def _grdc_metadata_reader(grdc_station_path, all_lines):  # noqa: C901
             attribute_grdc["altitude_masl"] = "NA"
 
         try:
-            attribute_grdc["dataSetContent"] = str(all_lines[20].split(":")[1].strip())
+            attribute_grdc["dataSetContent"] = str(all_lines[21].split(":")[1].strip())
```

Comment:
Would be nicer to extract this based on the actual header info/line start.
The line with this info seems to start with "# Data Set Content" which you should be able to find. Same for the other attributes.